box/resources/TestChainValidationMessageSuiteTestInitActorSequentialIDAddressCreate

```diff
@@ -1,2 +1,2 @@
-bafy2bzacecu7n7wbtogznrtuuvf73dsz7wasgyneqasksdblxupnyovmtwxxu,bafy2bzacebsjmkbe6lrpotwhk6zi7j7zllystyezo7fjerfqc5qorb5ziziv2,bafy2bzaceaxbu5r2sdn3tkaukzqya3m4y7bcrguohyzt7zqlxjtewlp4d5jo2,1918
```

Comment:
regenerated these files and go-filecoin has made some fixes since the previous generation.

Reply:
Please revert these or make a different PR that changes these values independent of big changes to the process that generates them.

Reply:
the format of these files has changes such that they only contain gas values. I have removed them since they are not required. I would prefer to change the files and the process for generating them all at once.

Reply:
If the format has changed, sure.